pkg/barbican/const.go

```diff
@@ -61,4 +61,8 @@ const (
 	PKCS11ClientDataVolume = "pkcs11-client-data"
 	// PKCS11DataVolume is the mount point used for PKCS11 client Data
 	PKCS11ClientDataMountPoint = "/var/lib/config-data/hsm"
+	// BarbicanUID - based on https://github.com/openstack/kolla/blob/master/kolla/common/users.py
```

Comment:
should we reference to tcib instead? https://github.com/openstack-k8s-operators/tcib/blob/main/container-images/kolla/base/uid_gid_manage.sh#L37

Reply:
oh yeah, it makes sense, updating the patch with the link you suggested! Thank you!